src/test/scala/sigmastate/utxo/BlockchainSimulationSpecification.scala

```diff
@@ -31,12 +31,13 @@ class BlockchainSimulationSpecification extends PropSpec
 
   import BlockchainSimulationSpecification._
 
-  def generateBlock(state: ValidationState, miner: ErgoLikeProvingInterpreter, height: Int): Block = {
+  def generateBlock(state: ValidationState, miner: ErgoLikeTestProvingInterpreter, height: Int): Block = {
     val minerPubKey = miner.dlogSecrets.head.publicImage
     val boxesToSpend = state.boxesReader.byHeightRegValue(height)
 
     val txs = boxesToSpend.map { box =>
-      val newBoxCandidate = new ErgoBoxCandidate(10, minerPubKey, Seq(), Map(heightReg -> LongConstant(height + windowSize)))
+      val newBoxCandidate =
+        new ErgoBoxCandidate(10, minerPubKey, Seq(), Map(heightReg -> LongConstant(height + windowSize)), height)
```

Comment:
@kushti I remember height register has Tuple type, but here it is Long

Reply:
@ergomorphic despite the name, it is just first non-mandatory register in the test, so the type could be any